src/pages/saksbehandling/revurdering/oppsummering/RevurderingsOppsummering.tsx

```diff
@@ -200,14 +204,19 @@ const RevurderingsOppsummering = (props: { sakId: string; revurdering: SimulertR
                         {sendtTilAttesteringStatus.error.body?.message}
                     </AlertStripeFeil>
                 )}
-                <div className={sharedStyles.knappContainer}>
-                    <Link className="knapp" to={forrigeURL}>
-                        {intl.formatMessage({ id: 'knapp.forrige' })}
-                    </Link>
-                    <Hovedknapp spinner={RemoteData.isPending(sendtTilAttesteringStatus)}>
-                        {intl.formatMessage({ id: 'knapp.sendTilAttestering' })}
-                    </Hovedknapp>
-                </div>
+                <RevurderingBunnknapper
+                    onNesteClick={() => {
+                        setSubmittedStatus(SubmittedStatus.NESTE);
+                        return formik.submitForm();
+                    }}
+                    nesteKnappTekst={intl.formatMessage({ id: 'knapp.sendTilAttestering' })}
+                    onTilbakeClick={() => {
+                        history.push(forrigeURL);
+                    }}
+                    onNesteClickSpinner={
+                        submittedStatus === SubmittedStatus.NESTE && RemoteData.isPending(sendtTilAttesteringStatus)
+                    }
```

Comment:
Ser for meg ut som at dette kan byttes ut med 
```suggestion
                    onNesteClickSpinner={RemoteData.isPending(sendtTilAttesteringStatus}
```

og at hele `SubmittedStatus`-enumen kan fjernes.